Containerfile

```diff
@@ -25,27 +25,9 @@ WORKDIR ${APP_ROOT}
 COPY ols ./ols
 COPY ./pyproject.toml ./pdm.lock ./
 
-RUN python3.11 -m venv ${APP_ROOT}/venv \
-    && source ${APP_ROOT}/venv/bin/activate \
-    # Install wheel for optimized dependency installation
-    # avoid using or leaving cache on final image
-    && pip install --no-cache-dir --upgrade pip pdm \
-    # Install project dependencies using PDM
-    && pdm install --frozen-lockfile \
-    # The following echo adds the unset command for the variables set below to the
-    # venv activation script. This is inspired from scl_enable script and prevents
-    # the virtual environment to be activated multiple times and also every time
-    # the prompt is rendered.
-    && echo "unset BASH_ENV PROMPT_COMMAND ENV" >> ${APP_ROOT}/venv/bin/activate
-
-RUN chown -R 1001:0 ${APP_ROOT} && \
-    chmod -R g+rx ${APP_ROOT}
```

Comment:
did you run this resulting image on a cluster to confirm we won't have permission issues w/o the chmod?

unfortunately nothing in our CI right now actually deploys the image on a cluster

Reply:
I plugged in this new image into the lightspeed-operator and indeed something is off:
```
[syedriko@fedora lightspeed-operator]$ oc -n openshift-lightspeed logs -f pod/lightspeed-app-server-5dd77b5757-b5mvd
Traceback (most recent call last):
  File "/usr/local/bin/uvicorn", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
...
  File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
ModuleNotFoundError: No module named 'ols'

```
I'll look some more into this.